vector/src/main/java/im/vector/app/features/roomdirectory/RoomDirectoryViewModel.kt

```diff
@@ -115,17 +115,10 @@ class RoomDirectoryViewModel @AssistedInject constructor(
 
     override fun handle(action: RoomDirectoryAction) {
         when (action) {
-<<<<<<< HEAD
             is RoomDirectoryAction.SetRoomDirectoryData -> Unit
-            is RoomDirectoryAction.FilterWith           -> filterWith(action)
-            is RoomDirectoryAction.JoinRoom             -> Unit
-            else                                        -> Unit
-=======
-            is RoomDirectoryAction.SetRoomDirectoryData -> setRoomDirectoryData(action)
             is RoomDirectoryAction.FilterWith -> filterWith(action)
             RoomDirectoryAction.LoadMore -> loadMore()
-            is RoomDirectoryAction.JoinRoom -> joinRoom(action)
->>>>>>> v1.4.27-RC2
+            is RoomDirectoryAction.JoinRoom -> Unit
```

Comment:
Why do we have a difference here? If the action is triggered, I think it can be caught, or is it caught in another place? If this action does not exist in Tchap, it should be never triggered, wdyt @yostyle ?